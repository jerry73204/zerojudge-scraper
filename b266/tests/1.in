6 8 1
9 4 9 1 3 3
0 4 9 6 6 5
6 3 1 9 0 8
9 2 5 8 2 6
0 3 1 2 0 5
8 8 2 5 1 6
5 6 3 5 5 5
7 0 7 0 3 6
0
