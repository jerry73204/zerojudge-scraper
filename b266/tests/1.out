6 8
3 5 8 6 5 6 5 6
3 6 0 2 0 1 5 3
1 6 9 8 2 5 5 0
9 9 1 5 1 2 3 7
4 4 3 2 3 8 6 0
9 0 6 9 0 8 5 7
